pkg/preflights/host-preflight.yaml

```diff
@@ -834,3 +835,18 @@ spec:
           - pass:
               when: "a-subnet-is-available"
               message: Specified CIDR is available.
+    - sysctl:
+        checkName: Sysctl Kernel Parameters
+        outcomes:
+          - fail:
+              when: 'net.ipv4.conf.default.arp_filter > 0'
+              message: "ARP filtering is enabled by default for newly created interfaces on the host. Disable it by running 'sysctl net.ipv4.conf.default.arp_filter=0'."
```

Comment:
I'm not sure these instructions make the change permanent. I think you might want to edit `/etc/sysctl.conf` and then use `sysctl -p`.

Reply:
True, changed the instructions in - https://github.com/replicatedhq/embedded-cluster/pull/1454/commits/6c800ba25f43a182f43b61f63ca7fe8f64c71444